tools/eval_metric.py

```diff
@@ -146,9 +161,9 @@ def eval_metric(args):
 
 
 @torch.no_grad()
-def evaluate(model, data_loader, coco_evaluator, device, print_freq):
+def evaluate(model, data_loader, coco_evaluator, device, print_freq, use_wandb, wandb_project, wandb_entity):
     # COCO evaluation
-    metric_logger = MetricLogger(delimiter="  ")
+    metric_logger = MetricLogger(use_wandb, wandb_project, wandb_entity, delimiter="  ")
```

Comment:
```suggestion
    metric_logger = MetricLogger(delimiter="  ", use_wandb=use_wandb, wandb_project=wandb_project, wandb_entity=wandb_entity)
```
Maybe we should adjust the order here.